app.py

```diff
@@ -88,3 +91,13 @@ def get_all_applicants(listing_id):
     """Gets all applicants from <listing_id>"""
     data = db.get_applicants(table_name="zap-applications", listing_id=listing_id)
     return data
+
+
+@app.route("/listings/{id}/toggle/visibility", methods=["PATCH"], cors=True)
```

Comment:
Are you planning to add other endpoints under /toggle? Structuring the route in this way seems redundant and potentially misleading.

Reply:
Not necessarily? I don't see toggle being used elsewhere, and each route should have it's own toggle function unless we make a generic toggle API that handles all cases - IMO that'd be over engineering and make things more complex. Usually, each business logic should be supported by its own API/functionality!